docs/tutorial.rst

```diff
@@ -29,113 +26,55 @@ The Yellowbrick library is a diagnostic visualization platform for machine learn
 About the Data
 --------------
 
-This tutorial uses a modified version of the mushroom dataset_ from
-the `UCI Machine Learning Repository <http://archive.ics.uci.edu/ml/>`__.
-Our objective is to predict if a mushroom is poisonous or edible based on
-its characteristics.
+This tutorial uses the mushrooms data from the Yellowbrick :doc:`api/datasets` module. Our objective is to predict if a mushroom is poisonous or edible based on its characteristics.
 
-.. _dataset: https://github.com/rebeccabilbro/rebeccabilbro.github.io/blob/master/data/agaricus-lepiota.txt
+.. NOTE:: The YB version of the mushrooms data differs from the mushroom dataset from the `UCI Machine Learning Repository <http://archive.ics.uci.edu/ml/>`__. The Yellowbrick version has been deliberately modified to make modeling a bit more of a challenge.
```

Comment:
This is a great addition to let users know what to expect prior to using our tutorial